src/api/Annotations.js

```diff
@@ -138,11 +162,14 @@ export default class Annotations extends MarkerBasedApi {
             return;
         }
 
+        const requestData = shouldFetchReplies ? { fields: 'replies' } : undefined;
+
         this.get({
             id: fileId,
             errorCallback,
             successCallback,
             url: this.getUrlForId(annotationId),
+            requestData,
```

Comment:
Have we tested all this new behavior and does this work as expected? I think we need to pass `params` here, instead.

Reply:
Thanks! Indeed we should pass `param` here

Reply:
Do we still need to update this to be `params`?

Reply:
[get method ](https://github.com/box/box-ui-elements/blob/master/src/api/Base.js#L239 )is receiving `requestData` property, although we need to have `params` property in it to fetch additional fields